docs/examples/changing-the-src-dir.md

```diff
@@ -0,0 +1,100 @@
+---
+description: |
+  Change the source directory to effectively manage your project.
+---
+
+When you initialize a project with `deno run -A -r https://fresh.deno.dev`,
+you'll end up with a project like the following:
+
+```
+.
+├── README.md
+├── components
+│   └── Button.tsx
+├── deno.json
+├── dev.ts
+├── fresh.gen.ts
+├── islands
+│   └── Counter.tsx
+├── main.ts
+├── routes
+│   ├── [name].tsx
+│   ├── api
+│   │   └── joke.ts
+│   └── index.tsx
+└── static
+    ├── favicon.ico
+    └── logo.svg
+```
+
+## Using a `src` directory
+
+If you'd like your code to live in a `src` directory (or any other place of your
+choosing), then you'll need to do the following things:
```

Comment:
```suggestion
If you'd like your code to live in an `src` directory (or any other directory of your
choosing), then you'll need to do the following things:
```

Reply:
Ah, the great debate around indefinite articles for abbreviations and acronyms. When I read text like "to live in a `src` directory", I read it as "... in a source directory". I guess you read it as "... in an S. R. C. directory" (i.e. each letter pronounced). I'll do what you say though, since I don't care too much and you're in charge.